environment.yml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - pytz
   - python
   - python-dateutil
-  - scipy
+  - scipy>=1.4.0
```

Comment:
The from_matrix and as_matrix methods were added in 1.4.0, the old as_dcm and from_csm methods were finally removed in 1.6.0